src/components/table/table.tsx

```diff
@@ -0,0 +1,239 @@
+    }
+  }
+
+
+  /* Sets the header of the table */
+  private setHeader(): any {
+    return <tr>
+      { (this.header.map((a) =>
+        <th>
+          <div class="col-lg-12">
+            <div class="row">
+              <div class="d-flex pr-5 bd-highlight">
+              <p class="mt-1">{a.description}</p>
+              </div>
+              <div>
+                <div class="row" onClick={() => this.sortData(a.key, "desc")}>
+                  <c-icon class="sort-desc" id={"desc" + a.key} name="scania-angle-down"></c-icon>
+                </div>
+                <div class="row" onClick={() => this.sortData(a.key, "asc")}>
+                  <c-icon class="sort-asc" id={"asc" + a.key} name="scania-angle-down"></c-icon>
+                </div>
+              </div>
+            </div>
+            <div class="row">
+              <div class="input-field col-md-12 pl-0">
+                <input id={"search" + a.key} onKeyUp={() => this.searchColumn()} type="text" class="form-control" />
+                <div id={"icon" + a.key} class="input-icon-filter"></div>
+              </div>
+            </div>
+          </div>
+        </th>
+    ))}
+      { this.hasEdit || this.hasDelete
+        ? <th></th>
+        : null
+      }
+    </tr>
+  }
+  
+  /* Sets the boby of the table */
+  private setbody(): any {
+    const keys = this.header.map(a => a.key);
+
+    return (this.filteredData.map((obj) =>
+        <tr>
+          { keys.map(key => <td>{obj[key]}</td>) }
+          { this.setDropDown(obj.rowIndex) }
+        </tr>
+    ));
+  }
+
+  /* Sets the dropdown options - current just exists edit and delete options */
+  /* TODO: Create a third option to add custom dropdown items */
+  private setDropDown(rowIndex: number): HTMLElement {
+    if(this.hasEdit || this.hasDelete)
+      return <td class="text-right">
+        <div class="btn-group">
+          <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Action
+          </button>
+          <div class='dropdown-menu dropdown-menu-right'>
+            { this.hasEdit && <a class="dropdown-item" onClick={ () => this.callbackDropdown("edit", rowIndex) }>Edit</a> }
+            { this.hasEdit && this.hasDelete? <div class="dropdown-divider"></div> : "" }
+            { this.hasDelete && <a class="dropdown-item text-danger" onClick={ () => this.callbackDropdown("delete", rowIndex) }>Delete</a> }
+
+          </div>
+        </div>
+      </td>
+  }
+  
+  /* Makes the callback for the developer */
+  private callbackDropdown(event: string, rowIndex: number)
+  {
+    debugger;
+    let obj = this.content[rowIndex];
+
+    if(event === "edit")
+      this.optEdit.emit(obj);
+
+    if(event === "delete")
+      this.optDelete.emit(obj);
+  }
+  
+  //
+
+  render() {
+      if(!this.isArray) {
+        return <div>
+            <h1>Wrong header/content format. Please contact Customer Support</h1>
```

Comment:
Can you change the text to:
Wrong header/content format. Please check documentation on https://digitaldesign.scania.com/components